db/app.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const app = express();
+const { getTopics } = require("./controllers/topics-controllers")
+
+app.get("/api/topics", getTopics)
+
```

Comment:
How is your invalid endpoint 404 test passing ? Where is the middleware function that deals with this?